config/config.go

```diff
@@ -1509,12 +1485,3 @@ func providedFlags(v *viper.Viper) map[string]interface{} {
 	}
 	return customSettings
 }
-
-// getRenamedKey returns the new key if it is set, otherwise it returns the deprecated key.
-func getRenamedKey(v *viper.Viper, deprecatedKey string, newKey string) string {
```

Comment:
This might be still useful to keep around if we deprecate keys in the future

Reply:
I think it would fail linting as dead code